modules/n1ql/pages/n1ql-language-reference/patternmatchingfun.adoc

```diff
@@ -384,7 +384,7 @@ REGEXP_SPLIT("/opt/couchbase/bin", "/") AS Unix;
 ]
 ----
 
-<1> The `REGEXP_SPLIT` function returns any zero-length matches that occur at the start of the _expression_ string, except when the split pattern is zero-length. Otherwise, it returns any zero-length matches immediately after a previous match.
+<1> The `REGEXP_SPLIT` function returns any zero-length matches that occur at the start of the _expression_ string, or immediately after a previous match.
```

Comment:
Leave as is

```suggestion
<1> The `REGEXP_SPLIT` function returns any zero-length matches that occur at the start of the _expression_ string, except when the split pattern is zero-length. Otherwise, it returns any zero-length matches immediately after a previous match.
```